pyproject.toml

```diff
@@ -90,6 +90,7 @@ dev = [
     "types-pkg_resources",
     # Security pin
     "gitpython>=3.1.30",
+    "ruamel.yaml",
```

Comment:
What is this for?

Reply:
This is used in the qm9 file?

Reply:
rewrote it with the default yaml implementation.